packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,74 @@ return [
 ];
 ```
 
+## Usage
+
+```php
+
+namespace App\Admin;
+
+use Runroom\SortableBehaviorBundle\Admin\AbstractSortableAdmin;
+
+class ExampleAdmin extends AbstractSortableAdmin
+{
+    // you can use SortableAdminTrait instead AbstractSortableAdmin
+
+    protected function configureListFields(ListMapper $list): void
+    {
+        $list
+        // ... some fields
+            ->add('_action', null, [
```

Comment:
There is a special const on Sonata, can you use them? https://github.com/Runroom/runroom-packages/blob/master/packages/basic-page-bundle/src/Admin/BasicPageAdmin.php#L77

Reply:
Just added a commit that uses them

Reply:
You are faster )